pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/utils/DriverUtils.java

```diff
@@ -217,4 +220,14 @@ public static boolean queryContainsLimitStatement(String query) {
     Matcher matcher = pattern.matcher(query);
     return matcher.find();
   }
+
+  public static String checkEnableNullHandling(PinotConnection connection, String query) {
+    if (query.contains(ENABLE_NULL_HANDLING)) {
+      return query;
+    }
+
+    return connection.isNullHandlingEnabled()
+      ? String.format("%s option(%s=true)", query, ENABLE_NULL_HANDLING)
```

Comment:
Let's use the new query option format:
`SET enableNullHandling = true; ...`

Reply:
Changed.